frontend/resourceadm/pages/DeployResourcePage/DeployResourcePage.tsx

```diff
@@ -266,19 +263,19 @@ export const DeployResourcePage = ({
                   }}
                   iconPlacement='left'
                   size='small'
-                  icon={<UploadIcon title='Lagre versjonsnummer' />}
+                  icon={<UploadIcon title={t('resourceadm.deploy_version_upload_button')} />}
                 >
-                  Last opp dine endringer
+                  {t('resourceadm.deploy_version_upload_button')}
                 </Button>
               </div>
             </div>
             <Label size='medium' spacing>
-              Velg miljø der du ønsker å publisere endringene
+              {t('resourceadm.deploy_select_env_label')}
             </Label>
             <div className={classes.deployCardsWrapper}>
               <ResourceDeployEnvCard
                 isDeployPossible={isDeployPossible('test', versionInTest)}
-                envName='Testmiljø TT-02'
+                envName={`${t('resourceadm.deploy_test_env')} TT-02`}
```

Comment:
Should  TT-02 be a part of the resourceadm.deploy_test_env? :)
```suggestion
                envName={`${t('resourceadm.deploy_test_env')}`}
```